### PR TITLE
Add image builder for NRRD files

### PR DESCRIPTION
**Pull request by nlessmann**
This adds an image builder for NRRD files. This file format also supports detached headers, similar to mhd/raw files - these are for now not supported because it is then complicated to determine the consumed files. Each header file can have multiple data files, which can be a list of files or a format string to identify the files. I think this is not used much anyway.

### CODE REVIEW

**Comment by jmsmkn** [comment]
I think those decisions make sense. About the test data - this is something you have constructed yourself and ok to add to the repo privacy/license wise?

File: panimg/image_builders/nrrd.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        if match is None:
+            raise InvalidNrrdFileError
+
+        # Check whether we support this version of the NRRD format
+        nrrd_version = int(match.group(1))
+        if nrrd_version < 1 or nrrd_version > 5:
+            raise InvalidNrrdFileError("File format not supported")
+
+        # Discard rest of preamble line
+        fp.readline(1000)
```
Comment (jmsmkn) [inline panimg/image_builders/nrrd.py:44]: Is the choice of 1000 bytes and lines arbitrary or is there something in the file spec on this?

Comment (nlessmann) [reply panimg/image_builders/nrrd.py:44]: That's an arbitrary value, I followed the implementation of header parsing function in metaio_utils.py in this (which actually uses 10,000, will adjust it to that)

**Comment by nlessmann** [comment]
> About the test data - this is something you have constructed yourself and ok to add to the repo privacy/license wise?

Those are derived from the .mha file with the same name in the test resources folder, I simply converted that file from mha to nrrd using ITK. I did the same before for the nifti image builder.

**Review by jmsmkn — APPROVED** [review]
LGTM, thanks!

**Pull request merged**